src/post_transform.c

```diff
@@ -222,21 +226,15 @@ int gen_reg_tile_file(PlutoProg *prog) {
   return 0;
 }
 
-/// Optimize the intra-tile loop order for locality and vectorization.
-int pluto_intra_tile_optimize_band(Band *band, int num_tiled_levels,
-                                   PlutoProg *prog) {
-  /* Band has to be the innermost band as well */
-  if (!pluto_is_band_innermost(band, num_tiled_levels, 0)) {
-    return 0;
-  }
+/// Returns the pointer to the best loop that can be moved to the innermost
+/// level. The loop is treated as vectorizable if 1) it is parallel 2) has
+/// stride 0 or stride 1 accesses (exhibits spatial locality). A weighted sum of
+/// these factors is taken into account. Any access that hinders spatial
+/// locality is peanalized.
```

Comment:
It's not clear why penalty is mentioned when you intend to say it should be all stride 0/1.